unison-cli/src/Unison/Codebase/Editor/HandleInput.hs

```diff
@@ -1939,6 +1933,46 @@ handleShowDefinition outputLoc inputQuery = do
             Nothing -> Just "scratch.u"
             Just (path, _) -> Just path
 
+-- Takes a maybe (namespace address triple); returns it as-is if `Just`;
+-- otherwise, tries to load a value from .unisonConfig, and complains
+-- if needed.
+resolveConfiguredGitUrl ::
+  PushPull ->
+  Path' ->
+  ExceptT (Output v) (Action' m v) WriteRemotePath
```

Comment:
I know you didn't introduce this comment, but since it's going to have your name on it going forward, you might want to improve it. 😅